test_security/CMakeLists.txt

```diff
@@ -256,7 +256,7 @@ if(BUILD_TESTING)
     set(ENV_PATH "$ENV{PATH}")
     file(TO_CMAKE_PATH "${ENV_PATH}" ENV_PATH)
     set(TEST_PATH "${ENV_PATH}")
-    if(rmw_implementation STREQUAL "rmw_connext_cpp")
+    if(rmw_implementation STREQUAL "rmw_connextdds")
```

Comment:
```suggestion
    if(rmw_implementation MATCHES "rmw_connext(.*)")
```

Reply:
Applied in [7069da0](https://github.com/ros2/system_tests/pull/463/commits/7069da00fed8ae1fd00ba97d108943df82f7319d).

Reply:
mmm didn't realize before but https://github.com/ros2/system_tests/pull/463/commits/7069da00fed8ae1fd00ba97d108943df82f7319d doesn't look correct, `STREQUAL` doesn't work like `MATCHES` so the `if()` is never satisfied.

No idea way, but CI is still happy with this, it seems that this `if()` is not more needed (?).

Reply:
Ha! My bad for being too hasty in making the "correction". Thank you for catching it!

Re: CI, maybe the directory is already added somewhere else?

Reply:
> Re: CI, maybe the directory is already added somewhere else?

maybe, can it be this https://github.com/rticommunity/rmw_connextdds/blob/b88131f28f7e6e6b0f5dff6eb9be133e512eccfd/rti_connext_dds_cmake_module/env_hook/rti_connext_dds.sh.in#L131? (or sth else that file is doing)

Anyways, it seems safer to keep the condition for the moment.

Reply:
> maybe, can it be this https://github.com/rticommunity/rmw_connextdds/blob/b88131f28f7e6e6b0f5dff6eb9be133e512eccfd/rti_connext_dds_cmake_module/env_hook/rti_connext_dds.sh.in#L131? (or sth else that file is doing)
> 

It's possible, although I would have expected `RTI_OPENSSL_BIN` to point to something other than `$NDDSHOME/bin`. It ultimately comes down to what the value of that variable is.

> Anyways, it seems safer to keep the condition for the moment.

Agreed.